test.py

```diff
@@ -1,2 +1,4 @@
 def main():
     pass
+   asd
+   
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/test.py
+++ b/test.py
@@ -1,4 +1,4 @@
 def main():
     pass
    asd
-   
+
```